warehouse/README.md

```diff
@@ -43,6 +46,7 @@ are already configured/installed.
          schema: <yourname>
          threads: 8
          timeout_seconds: 3000
+         maximum_bytes_billed: 100000000000
```

Comment:
What happens if you run initial setup with this set? This is not enough for initial build, right? Do only the large tables fail?

Reply:
Yes, I'll bump to 2 TB so most things (except trip updates) will fail. A full `dim_stop_times` is ~1.1 TB right now.

Reply:
Our lookback for trip updates is about 1 TB, so 2 will work for awhile.